Cargo.lock

```diff
@@ -4403,6 +4404,25 @@ dependencies = [
  "sp-std",
 ]
 
+[[package]]
+name = "manta-vesting"
+version = "3.0.6"
```

Comment:
You should merge manta-pc, it has the 3.0.8 release already.

Reply:
I merged latest manta-pc, it's 3.0.6. I think you didn't align it in your pr. But it's fine, I will bump version to 3.0.9.